src/app_engine/apprtc.py

```diff
@@ -215,18 +216,20 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
   hd = request.get('hd').lower()
   if hd and video:
     message = 'The "hd" parameter has overridden video=' + video
-    logging.error(message)
-    error_messages.append(message)
+    logging.warning(message)
+    # HTML template is UTF-8 while python defaults to unicode.
```

Comment:
This comment makes no sense. UTF-8 is an encoding of all the chars in unicode standard.